package/MDAnalysis/lib/distances.py

```diff
@@ -48,9 +48,12 @@
 
    "OpenMP"   :mod:`c_distances_openmp` parallel implementation in C/Cython
                                         with OpenMP
-   "distopia"  `_distopia`               SIMD-accelerated implementation
+
+   "distopia"  `_distopia`              SIMD-accelerated implementation
+                                        with the distopia_ library
```

Comment:
```suggestion
                                        with the `distopia`_ library
```
?

Reply:
Both work but yours is more explicit 👍 – thx

Reply:
TIL